docs(stripe): adding docs for stripe (#863)
front-commerce/developers.front-commerce.com: docs/03-extensions/stripe/how-to/disable-automatic-capture.mdx

Reviewer: **Issue:** this is a lot of docs to explain how to add an environment variable?

Implement this in the Environment Variables docs: https://deploy-preview-863--heuristic-almeida-1a1f35.netlify.app/docs/3.x/extensions/stripe/reference/environment-variables

Author: It has it's dedicated page because it's in the acceptance criterias for the issue, we can maybe enhance this section later

---
title: Disable automatic capture
description:

>   "This guides explains how to disable Automatic capture of Stripe payments"

Reviewer 2: ```suggestion
  "This guide explains how to disable Automatic capture of Stripe payments."
```

Author: fixed

sidebar_position: 3
---

<p>{frontMatter.description}</p>

Optionally the environment variable `FRONT_COMMERCE_STRIPE_DISABLE_CAPTURE` can
be set to `true` to configure the Stripe module so that the payment is not

> automatically captured. If you want this behavior, you have to change your
> `.env` in the following way:

Reviewer: ```suggestion
automatically captured. You can opt-in to this behavior, by adding the variable to your
`.env`:
```

Author: fixed
